components/builder-web/app/actions/origins.ts

```diff
@@ -163,6 +165,22 @@ export function inviteUserToOrigin(username: string, origin: string, token: stri
     };
 }
 
+export function fetchOriginsPackageCount(origins) {
+    return dispatch => {
+        origins.forEach(origin => {
+            depotApi
+                .getUnique(origin)
```

Comment:
This can be updated to use the package_stats API now:

`/v1/depot/pkgs/origins/:origin/stats`